Sources/snowman/main.swift

```diff
@@ -2,11 +2,17 @@ import SnowManLogic
 import SnowManDictionary
 
 let dictionaryManager = DictionaryManager(words: SnowManDictionary.words) 
-let guessManager = GuessManager(totalIncorrectGuessesAllowed: 10)
 let word = dictionaryManager.getRandomWord()
 
 let io = IO()
+
 let view = View(io: io)
+
+let maxNumberOfGuesses = 10
+view.promptNumberOfGuesses()
+let numberOfGuesses = view.receiveNumberOfGuesses(maxNumberOfGuesses)
```

Comment:
@mnewsome Half yes, half no. I removed the `maxNumberOfGuesses` requirement, so that eliminates line 11. I moved the `promptNumberOfGuesses()` function to be initialized when the `View` class is created. Not sure how to avoid line 13, though, if I am trying to use dependency injection to pass this into the `GuessManager` class instance. I need the `View` to call the method, I need the method to be injected into `GuessManager`, and I need both `View` and `GuessManager` in main to be passed into `Game`. I could eliminate that line by doing something silly like this:
```
let guessManager = GuessManager(numberOfGuesses: Int(view.receiveNumberOfGuesses()))
```
But I'm not sure that this is solving the problem ;-)